traffic_ops_ort/testing/docker/traffic_ops/Dockerfile

```diff
@@ -35,12 +35,16 @@ RUN set -o nounset -o errexit && \
 	if [[ "${RHEL_VERSION%%.*}" -eq 7 ]]; then \
 		use_repo=''; \
 		enable_repo=''; \
+		# needed for llvm-toolset-7-clang, which is needed for postgresql13-devel-13.2-1PGDG, required by TO rpm
+		dnf -y install centos-release-scl-rh; \
```

Comment:
The error mentioned above relating to `devtoolset-7-gcc` and `install_goose.sh` happens in this Dockerfile, too.